src/main/scala/org/nbastats/model/Bio.scala

```diff
@@ -0,0 +1,5 @@
+package org.nbastats.model
+
+case class Bio(name: String, age: Age, positions: Seq[String], experience: Int)
```

Comment:
we can have a `sealed trait` with enumerated values to represent positions, since there are only 5 of them

```scala
sealed trait Position
case object PointGuard extends Position
case object ShootingGuard extends Position
case object SmallForward extends Position
case object PowerForward extends Position
case object Center extends Position
```

Reply:
I'll go ahead and add this into the Bio.scala file